packages/muelu/adapters/xpetra/MueLu_RefMaxwell_def.hpp

```diff
@@ -372,6 +372,7 @@ namespace MueLu {
         coarseLevel.Set("R",R11_);
         coarseLevel.Set("Coordinates",CoordsH_);
         coarseLevel.Set("number of partitions", numProcsAH);
+        coarseLevel.Set("repartition: heuristic target rows per process", 1000);
```

Comment:
@cgcgcg I thought the idea was that this should be the same as `repartition: target rows per processor`?

Reply:
Yes, but that variable gets transformer depending on the number of threads. We should probably rename this one though.

Reply:
I see.  If someone sets `repartition: min rows per thread` to a positive value, then this would in turn modify `repartition: target rows per proc`.  I don't object to your change, but this does make the number of options rather daunting.

Reply:
We are not exposing this parameter to the user, so I think this is ok. If the user does not want the default, he/she needs to set `mj_premigration_coordinate_count`, same as before.